libnetwork/drivers/overlay/ovmanager/ovmanager.go

```diff
@@ -65,7 +65,7 @@ func Register(r driverapi.DriverCallback, config map[string]interface{}) error {
 		config:   config,
 	}
 
-	d.vxlanIdm, err = idm.New(nil, "vxlan-id", 0, vxlanIDEnd)
+	d.vxlanIdm, err = idm.NewWithNoStore("vxlan-id", 0, vxlanIDEnd)
```

Comment:
```suggestion
	d.vxlanIdm, err = bitmap.New(vxlanIDEnd)
```
With no store, the ID is unused. Without a start offset, `idm` is functionally indistinguishable from a bitmap. So just use a bitmap directly!